apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -112,6 +118,14 @@ type ClusterQueueSpec struct {
 	StopPolicy *StopPolicy `json:"stopPolicy,omitempty"`
 }
 
+type AdmissionCheckStrategy struct {
```

Comment:
I'm thinking if the strategy should be a struct that wraps the list of admission checks:
```yaml
admissionChecksStrategy:
   admissionChecks:
    - name1
      onFlavors: [a, b, c]
```
The reason being to allow for introducing properties common for all admission checks, analogously as we do for PodFailurePolicy or SuccessPolicy in Job. One example I would see this useful might be to say if the admission checks are "parallel" or "sequential".

WDYT @alculquicondor @tenzen-y ?

Reply:
+1
It could be useful.

Reply:
+1
Thank you for pointing this out.

Reply:
+1 I like this idea, I've added the changes